analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Mutex.cs

```diff
@@ -350,5 +350,19 @@ public void MutextAquireByConstructor_SwitchExpression(int x)
                 m.ReleaseMutex();
             }
         }
+
+        public void NullTrackingCondition()
```

Comment:
This issue is not related to https://github.com/SonarSource/sonar-dotnet/issues/5522. Did you open an issue for this case?